pkg/kubelet/cm/dra/manager.go

```diff
@@ -333,6 +330,7 @@ func (m *ManagerImpl) UnprepareResources(pod *v1.Pod) error {
 			// before this checkpoint succeeds, we will simply call into this (idempotent)
 			// function again.
 			claimInfo.deletePodReference(pod.UID)
+			m.cache.add(&claimInfo)
```

Comment:
Again -- `add()` as a function name seems weird here.

Reply:
Agree, but having two identical functions - `add` and `update` doesn't make much sense either. We should come up with one name for both.  My proposal is `addOrUpdate`.